lib/network.go

```diff
@@ -2409,7 +2409,7 @@ func DecodeHeaderVersion2(rr io.Reader) (*MsgDeSoHeader, error) {
 	}
 
 	// TstampNanoSecs
-	retHeader.TstampNanoSecs, err = ReadUvarint(rr)
+	retHeader.TstampNanoSecs, err = ReadVarint(rr)
```

Comment:
Yeah can we change this back? I know it's kinda unfortunate that we can't just wipe and move everything to Int64, but this makes me a bit nervous.

Reply:
Header version is newly added with PoS. This decoding is safe! Or are you concerned with something else

Reply:
Thanks for clarifying! I forgot that this was new, so yeah this is good then!